packages/sanity/src/core/comments/hooks/useCommentsUpsell.ts

```diff
@@ -3,10 +3,6 @@ import {CommentsUpsellContext} from 'sanity/_singletons'
 
 import {type CommentsUpsellContextValue} from '../context/upsell/types'
 
-/**
```

Comment:
any particular reason this was removed?

Reply:
Its not exported at the package level and never was I think, so un-needed. This prompted me to double check the original export list:

CommentInput,
  type CommentIntentGetter,
  COMMENTS_INSPECTOR_NAME,
  CommentsEnabledProvider,
  CommentsIntentProvider,
  type CommentsIntentProviderProps,
  CommentsProvider,
  useCommentsEnabled,


  type CommentsIntentProviderProps was still missing so I've re-added it